style.css

```diff
@@ -522,10 +514,24 @@ q:after {
 	text-rendering: auto;
 }
 
+/* Typography for Thai Font */
+
+html[lang="th"] h1, h2, h3, h4, h5, h6 {
```

Comment:
@imnok Yes, you can edit the file and commit again. @melchoyce mentioned about not removing the spaces after the paragraphs. It is better to make such a change in a separate pull request if it is still needed.